drivers/interrupt_controller/intc_gicv3_priv.h

```diff
@@ -67,7 +67,7 @@
 #define GICR_TYPER_AFFINITY_VALUE_MASK		0xFFFFFFFFUL
 #define GICR_TYPER_AFFINITY_VALUE_GET(_val)	MASK_GET(_val, GICR_TYPER_AFFINITY_VALUE)
 #define GICR_TYPER_LAST_SHIFT			4
-#define GICR_TYPER_LAST_MASK			0x10UL
+#define GICR_TYPER_LAST_MASK			0x1UL
```

Comment:
I see no usage of `GICR_TYPER_LAST_MASK`?

Reply:
Also, according to the documentation the `LAST` bit for `GICR_TYPER` is on bit `[4]` so it is correct to have the mask as `BIT(4) == 0x10UL`.

Reply:
Hi Carlo,

Here: https://github.com/zephyrproject-rtos/zephyr/blob/b8f7c81dd3930ae1dde60dcf1daba32d03176816/drivers/interrupt_controller/intc_gicv3.c#L559

Reply:
oh, great, this file is basically subverting the usual meaning of the term `MASK`, even adding a new `MASK()` macro to switch between the canonical meaning and the special one, wonderful.